gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/store/GemFireStore.java

```diff
@@ -2139,6 +2139,9 @@ private void clearStatics(boolean forBoot) {
       // Instead do this at the boot time itself
       CallbackProcedures.clearStatics();
       DistributionObserver.clearStatics();
+      if (!forBoot) {
+        FabricDatabase.TEST_INDEX_RECREATE = false;
+      }
```

Comment:
Ok. Let me think. I will go ahead and do the check-in and think of a cleaner way. Since the IRF file is a kind of a binary file so it will be difficult to change but yes ofcourse an older irf file can replace the new one and then we can test this.